ts-boilerplate-common/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "module": "commonjs",
+      "target": "es5",
+      "outDir": "dist",
+      "rootDir": "./src",
+      "strict": false,

```

Comment:
Why disable strict mode by default? I think as a boilerplate it'd be better to promote best practices, and strict mode enable safer compilation defaults and better potential error catching.